components/cosmetic_filters/resources/data/element_picker.ts

```diff
@@ -6,23 +6,18 @@ const NSSVG = 'http://www.w3.org/2000/svg'
 
 let pickerDiv: HTMLDivElement | null
 let shadowRoot: ShadowRoot | null
+const isAndroid = /(android)/i.test(navigator.userAgent)
```

Comment:
is this best practice? Not a big deal, just a bit suprising, imo

Reply:
Changed to use initialization call which returns the platform